### PR TITLE
Reduce Operator & Console replicas to 2 each

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Per discussion in slack @spadgett

### CODE REVIEW

**Comment by spadgett** [comment]
I'd suggest updating the example/cr.yaml as well in this pr (the replica count and comment). Then we don't need #101.

File: pkg/console/operator/operator.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ const (
 )
 
 const (
-	minimumConsoleReplicas = 3
+	minimumConsoleReplicas = 2
```
Comment (spadgett) [inline pkg/console/operator/operator.go]: Should this just be called `consoleReplicas` since it will always be that value (at least for now).

**Comment by benjaminapetersen** [comment]
Agreed, I was going to close #101.

File: pkg/console/operator/operator.go [hunk 2]
```diff
@@ -251,7 +251,7 @@ func (c *consoleOperator) defaultConsole() *consolev1alpha1.Console {
 				// if Verison is not 4.0.0 our reconcile loop will not pick it up
```
Comment (jhadvig) [inline pkg/console/operator/operator.go]: Not new, but `r/Verison/Version`

Comment (benjaminapetersen) [reply pkg/console/operator/operator.go]: Yeah lets polish that stuff up.

**Comment by spadgett** [comment]
/retest

```
Pulling image "registry.svc.ci.openshift.org/ci/clonerefs@sha256:a3d5b45cfc3a195d26231e30b824d9c086a97f5735dcc208f26dc50c2a1e5a75" ...
error: error copying source path /app/prow/cmd/clonerefs/app.binary.runfiles/test_infra/prow/cmd/clonerefs/linux_amd64_pure_stripped/app.binary to .: API error (404): {"message":"lstat /var/lib/docker/overlay2/61cccf6897d68f87ea7a68030200add65b2ddec4e2c11a09b76a44edd1cd840c/merged/app/prow/cmd/clonerefs/app.binary.runfiles/test_infra/prow/cmd/clonerefs/linux_amd64_pure_stripped/app.binary: no such file or directory"}
```

**Pull request merged**